AmplifyPlugins/Predictions/CoreMLPredictionsPlugin/Dependency/CoreMLSpeechBehavior.swift

```diff
@@ -6,8 +6,9 @@
 //
 
 import Foundation
-import AWSTranscribe
+import Amplify
 
-protocol AWSTranscribeBehavior {
-    func getTranscribe() -> AWSTranscribe
+protocol CoreMLSpeechBehavior: class {
```

Comment:
weird its showing up this way, this is a new file I made not a changed file that had a protocol in it